Makefile

```diff
@@ -112,3 +113,6 @@ docker: Dockerfile Cargo.lock
 
 .PHONY: all
 all: build test
+
+print-%:
```

Comment:
Is this needed?

Reply:
Sorry, just a forgotten debug rule. Thanks for pointing that out.